packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -113,9 +120,13 @@ export default {
                   { who: 'j4RxTMa1QVucodYPfQGA2JrHxZP944dfJ8qdDDYKU4QbJCWNP', value: joy(0.2) },
                   { who: 'j4Rxkb1w9yB6WXroB2npKjRJJxwxbD8JjSQwMZFB31cf5aZAJ', value: joy(0.2) },
                   { who: 'j4RyLBbSUBvipuQLkjLyUGeFWEzmrnfYdpteDa2gYNoM13qEg', value: joy(0.2) },
+                  { who: 'j4S998Thq5kQHyurofh8QfHrcFN2c1T19gTdMGUVVx5EHKgky', value: joy(0.2) },
                 ],
               },
               erasTotalStake: joy(130_000),
+              slashingSpans: {
+                size: Math.floor(Math.random() * 2),
```

Comment:
For this mock to work `withUnwrap` could be redefined like this:
```ts
const withUnwrap = (data: Record<any, any>) =>
  Object.defineProperties(data, {
    unwrap: { value: () => data },
    isSome: { value: Object.keys(data).length > 0 },
  })
```
This way defining an empty object would simulate a Nothing.